src/ui/Templates/Template.ts

```diff
@@ -246,11 +246,14 @@ export class Template implements ITemplateProperties {
     return element;
   }
 
-  private makeTemplateAccessible(template: HTMLElement, result: IQueryResult) {
-    template.tabIndex = 0;
-    template.setAttribute('role', 'heading');
-    template.setAttribute('aria-level', '2');
-    template.setAttribute('aria-label', `Result`);
-    result && result.title && template.setAttribute('title', result.title);
+  private makeFirstResultLinkAccessible(template: HTMLElement) {
+    const resultLink = $$(template).find('.CoveoResultLink');
```

Comment:
I faced the circular dependency issue in trying to reference the ResultLink id. I could put the id in a separate file like I did on the aria-live PR, but how do you feel about creating these additional files with one line? I am now thinking maybe hard-coding is ok ...

Reply:
Hum, why does it need to be done here ?
Why not simply moving that to the actual `CoveoResultLink` component ?

It's extremely weird to be modifying the component rendering/attributes outside of it's module for pretty much no reason ?